src/definitions/elements/button.less

```diff
@@ -169,19 +169,24 @@
 
     box-shadow: 0 0 0 1px transparent;
   }
-}
-& when (@variationButtonLabeledIcon){
-  .ui.labeled.icon.loading.button .icon {
+  .ui.ui.loading.button .label {
```

Comment:
.ui.ui.loading.button

.ui.ui <-- is this a typo?

Reply:
This is a CSS technique to increase specificity of the selector
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
I even had to increase it again to make it work properly when compiled by 6d249925aa0695b344c5f2e92417f8b76ed12237 :wink: